src/utils/media-utils.js

```diff
@@ -71,5 +73,11 @@ export const addMedia = (src, resize = false) => {
         entity.setAttribute("media-loader", { src: "error" });
       });
   }
+
+  entity.addEventListener("media_resolved", ({ detail }) => {
```

Comment:
The entity's `media-loader`'s `src` can change, which may mean that this `contentOrigin` is no longer correct. So far it will be, but we should be aware of this. PDF's are the only thing that makes use of the changing `src` right now AFAIK

Reply:
Hmm, I think contentOrigin is independent of src. In this context, contentOrigin is a value from the ObjectContentOrigins enum. i.e. one of URL, FILE, CLIPBOARD or SPAWNER.